Source/DomainObjects/AuthState.swift

```diff
@@ -60,7 +60,7 @@ public extension AuthStatus {
         }
     }
     
-    public var rawValue: String {
```

Comment:
why? this could be a breaking change

Reply:
This is unneeded, since the whole extension is marked as public. Also this causes compilation warning: "'public' modifier is redundant for property declared in a public extension".